hs_core/0058_solrqueue.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this one can be removed, we should only need the one in the migrations folder.